### ISSUE TITLE
Vec and Map Debug impls include Result Oks in output

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/rs-stellar-contract-sdk#283]
Example:
```
---- test::test_scval_accessibility_from_udt_types stdout ----
thread 'test::test_scval_accessibility_from_udt_types' panicked at 'assertion failed: `(left == right)`
  left: `UdtStruct { a: 10, b: 12, c: Vec(Ok(Map(Ok((10, Symbol(10))), Ok((5, Symbol(5)))))) }`,
 right: `UdtStruct { a: 10, b: 12, c: Vec() }`', tests/udt/src/lib.rs:76:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by leighmcculloch** [comment]
This is not a bug. The iterators for the types return Result's appropriately because type conversion can fail. It is reasonably important the Debug output includes this for the case that conversion fails on an entry.
